README.md

```diff
@@ -1,2 +1,20 @@
 # Sandbox CDS Services
```

Comment:
Add a **License** section to the bottom of this README.md:

```
   Copyright 2017 Cerner Innovation, Inc

   Licensed under the Apache License, Version 2.0 (the "License");
   you may not use this file except in compliance with the License.
   You may obtain a copy of the License at

       http://www.apache.org/licenses/LICENSE-2.0

   Unless required by applicable law or agreed to in writing, software
   distributed under the License is distributed on an "AS IS" BASIS,
   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   See the License for the specific language governing permissions and
   limitations under the License.
```

Also, create a `NOTICE` file with the exact same contents as the above text.